Utils/action_utils_getcurrenttime.json

```
@@ -40,9 +46,7 @@
         "default": "UTC 0"
       }
     },
-    "required": [
-      "selectedTimezone"
-    ]
+    "required": []
```

We can use `AnyOf` to require `selectedNamedTimezone` or `selectedTimezone` (like in [MicrosoftDefender action](https://github.com/SEKOIA-IO/automation-library/blob/main/MicrosoftDefender/action_update_alert.json#L8))
```suggestion
    "anyOf": [
       {
           "required": ["selectedTimezone"]
       },
       {
           "required": ["selectedNamedTimezone"]
       }
    ]
```